### ISSUE TITLE
"Es konnte keine Verbindung zum Plugin-Verzeichnis hergestellt werden. Stellen Sie sicher, dass der Server Verbindung mit www.dokuwiki.org aufnehmen darf und überprüfen Sie ihre Proxy-Einstellungen."

### ISSUE THREAD

**Issue by jens62** [splitbrain/dokuwiki#3855]
I get the message in the headline when i try to manage the plugins.

When I run the command below on my server, it reports a "certificate has expired" error.

```
# curl "https://www.dokuwiki.org/lib/plugins/pluginrepo/api.php"
curl: (60) SSL certificate problem: certificate has expired
...
```

Might that be the cause of the 'repo_error'?

**Comment by splitbrain** [comment]
Certificate on www.dokuwiki.org looks good to me. Expires March 25th 2023. Maybe something is man-in-the-middle attacking you? Corporate firewall?

**Comment by jens62** [comment]
Andreas,
thanks for the fast reply.

Indeed your answers helps.

On another machine in my network the curl command works as expected.

My server (Qnap-NAS) does not trust the certificate. (Private installation, no firewall issue).

The problem is on my end. I am going to investigate the server.

BR
Jens

**Comment by jens62** [comment]
I am running dokuwiki on a qnap server.
In order to nail down the issue a run the curl command above and just trusted the message from curl "certificate has expired".
Because of the (fast!!) reply of Andreas, I tested on other machines with success. I.e. actually no certificate issue.

Finally I searched the internet again and found a solution from Andreas:

[The plugin repository could not be contacted](https://forum.dokuwiki.org/d/20344-the-plugin-repository-could-not-be-contacted/7)

This solved the issue for me.
